components/layout_2020/display_list/stacking_context.rs

```diff
@@ -64,16 +65,29 @@ pub(crate) struct StackingContext<'a> {
 
     /// All float pseudo stacking context children of this stacking context.
     float_stacking_contexts: Vec<StackingContext<'a>>,
+
+    /// CSS filters to be applied to this stacking context (including opacity).
+    filters: Vec<wr::FilterOp>,
+
+    /// The blend mode with which this stacking context blends with its backdrop.
+    mix_blend_mode: wr::MixBlendMode,
```

Comment:
Do these need to be kept in the stacking context tree, or could they be recomputed from the `&ComputedValues` style of the element that generates this stacking context?

Reply:
I think they can be recomputed. I can try to organize the code in that way.

Reply:
Okay. I've pushed a new version of this change that minimizes data members in the `StackingContext` data structure.